src/index.js

```diff
@@ -1,25 +1,139 @@
 import './style.css';
-import { displayListItem, inputField } from './todoListSelector.js';
 
-const addtodoItem = (todovalue) => {
-  const todoContainer = document.createElement('div');
-  todoContainer.classList.add('todoContainer');
-  todoContainer.innerHTML = `
-            <input type="checkbox" class="checkbox">
-            <span>${todovalue}</span>
-            <i class="fas fa-ellipsis-vertical"></i>
-            <i class="fas fa-trash"></i>
-            <hr>
+class MyObjects {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+let array = [];
+const todoList = () => {
+  const form = document.querySelector('.form');
+  const list = document.createElement('div');
+  list.className = 'input-div';
+  form.appendChild(list);
+  const checkboxes = document.createElement('input');
+  checkboxes.className = 'input';
+  checkboxes.type = 'checkbox';
+  const listText = document.createElement('p');
+  listText.className = 'listContent';
+  const editBtn = document.createElement('i');
+  editBtn.className = 'fas fa-ellipsis-v';
+  const deleteBtn = document.createElement('i');
+  deleteBtn.className = 'fas fa-trash-alt icon2';
+  list.append(checkboxes, listText, editBtn, deleteBtn);
+
+  checkboxes.addEventListener('click', () => {
+    editBtn.classList.toggle('remove-icon-active');
+    deleteBtn.classList.toggle('icon2');
+    listText.classList.toggle('listContent-disable');
+    list.classList.toggle('changeBg');
+    const getting = JSON.parse(localStorage.getItem('list'));
+    const empty = [];
+    const selectInput = document.querySelectorAll('.input-div');
+    for (let i = 0; i < getting.length; i += 1) {
+      if (selectInput[i].classList.contains('changeBg')) {
+        getting[i].completed = true;
+      } else {
+        getting[i].completed = false;
+      }
+      empty.push(getting[i]);
+      localStorage.setItem('list', JSON.stringify(empty));
+    }
+  });
+  const clearAll = document.querySelector('#clear');
+  clearAll.addEventListener('click', () => {
+    const getting = JSON.parse(localStorage.getItem('list'));
+    const variable = document.querySelectorAll('.changeBg');
+    for (let i = 0; i < variable.length; i += 1) {
+      form.removeChild(variable[i]);
+    }
+    const empty = [];
+    for (let i = 0; i < getting.length; i += 1) {
+      if (getting[i].completed === true) {
+            continue; // eslint-disable-line
+      }
+      empty.push(getting[i]);
+    }
+    localStorage.setItem('list', JSON.stringify(empty));
+  });
 
-        `;
+  deleteBtn.addEventListener('click', () => {
```

Comment:
- [ ] Good work implementing the edit button/functionality, However, I can't remove the task, please implement this so your app has the functionality for removing tasks (one by one - not just clear all)
![Screenshot_20220610_211739](https://user-images.githubusercontent.com/25227022/173145573-142d7a82-fc6e-445d-8dc2-0230909d8020.png)